Arnold AA_seed support

johnhaddon opened the pull request:
This causes Arnold's AA_seed parameter to be set to the current frame number by default, with the option of overriding it using a setting on the ArnoldOptions node.

andrewkaufman commented on src/GafferAppleseed/IECoreAppleseedPreview/Renderer.cpp:
```diff
@@ -1576,6 +1577,12 @@ class AppleseedRenderer : public IECoreScenePreview::Renderer
 
+			if( name == g_frameOptionName )
+			{
+				/// \todo Does this have a meaning in Appleseed?
```
Is this a merge as is todo, or a wait for @est77 to inform us how to implement before merging?

johnhaddon replied:
It'd be great to get Est's input, but I don't think it needs to block a merge.

est77 replied:
We don't have a setting for the current frame, but I was thinking now that we can add one and then I can fill the details here in a future PR.

andrewkaufman commented:
Needs a rebase

danieldresser commented:
The only thing I wasn't sure about here was whether there should be a round() in "new IntData( Context::current()->getFrame() ) );"?

johnhaddon commented:
Rebased and added @danieldresser's suggested rounding - I think that was the right suggestio, because it matches what we do to convert from the floating point frame to the integer frame in #### substitutions.
